File: TrainingJSPIV/src/models/UserModel.java
package models;

public class UserModel {
	private Long id;
	private String name;
	private String password;
	
	public UserModel () {}
	
	public UserModel (Long id, String name, String password) {
		this.id = id;
		this.name = name;
		this.password = password;
	}
	
	public Long getID() {
		return this.id;
	}
	
	public String getName () {
		return this.name;
	}
	
	public String getPassword () {
		return this.password;
	}
}

File: PHP/laravel-with-database/app/Models/UserModel.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class UserModel extends Model
{
    protected $table = 'user';

    protected $fillable = [
        'name',
        'email',
        'password',
        'remember_token'
    ];

    public function sales () {
        return $this->belongsTo(salesModel::class, 'id_product');
    }

    public function course () {
        return $this->belongsToMany(coursesModel::class, 'user_courses', 'user_id', 'courses_id');
    }

    use HasFactory;
}
